### ISSUE TITLE
Self-Service - Warn about missing admin privileges if using agent but --run-actual is passed

### ISSUE THREAD

**Issue by pascalberger** [chocolatey/chocolatey-licensed-issues#16]
### What You Are Seeing?
If I have background agent running Chocolatey no longer warns if executed without administrative privileges, which makes sense. 

### What is Expected?
If `choco install` is run with the `--run-actual` parameter choco should warn about missing administrative privileges, even if background mode is active.

### How Did You Get This To Happen? (Steps to Reproduce)
- Enable background agent
- Run `choco install somepackage --run-actual`

**Comment by ferventcoder** [comment]
I'm guessing you followed the instructions at https://chocolatey.org/docs/features-agent-service#background-mode-setup and ran:

`choco feature disable -n showNonElevatedWarnings`

**Comment by ferventcoder** [comment]
This turns it off for all. Are you wanting a feature like this: `showNonElevatedWarningsToAdmins` (or whatever is consistent with naming)?

**Comment by pascalberger** [comment]
Yes, I have run `showNonElevatedWarnings` disabled. Not sure if `showNonElevatedWarningsToAdmins` is what I want. I would expect Chocolatey to not warn about elevation if using background service, except if `--run-actual` parameter is passed to the command. So something like `disableElevatedWarningsIfBackgroundModeIsUsed` might be more appropriate.

**Comment by ferventcoder** [comment]
@pascalberger but you will be able to not route through background service when running choco as an administrator (read: with or without elevation) with https://github.com/chocolatey/chocolatey-licensed-issues/issues/14. So the name of the feature should make sense to explain that it would only show that warning to admins if they are in a non-elevated status.

**Comment by ferventcoder** [comment]
Fixed the wording above

**Comment by pauby** [comment]
`--run-actual` is an undocumented option that should not be used for end-user installations.

I'm going to go ahead and close this, but we can reopen it again if necessary.
